modules/gz-math/20241112.1/presubmit.yml

```
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
```

```suggestion
    bazel: ${{ bazel }}
    build_flags:
    - '--cxxopt=-std=c++17'
```
since gz-math uses C++17 features such as nested-namespace-definition

---

That should fix some of the macos presubmit failures.

---

Same for gz-utils and sdformat presubmits

---

Done.